### ISSUE TITLE
How to get the accuracy of train_dataset and val_dataset and display in tensorboard?

### ISSUE THREAD

**Issue by dingyan-dy** [matterport/Mask_RCNN#2935]
I use maskrcnn model in train,but I also want to get some metrics process of train and val, such as map、miou、acc. I don't know how to use custom_callback to finish it. So if you can tell me some method to use the custom_callback.

**Comment by ydzat** [comment]
hello, have you already solved this problem? I want to know too.
